### PR TITLE
Added files under each task page and created tabular-classification task page

### PR DESCRIPTION
**Pull request by merveenoyan**
FYI @julien-c @osanseviero

### CODE REVIEW

**Comment by osanseviero** [comment]
FYI when we discussed about default task page we said we could have a single tasks/src/default/about.md and data. That file would be a simple markdown with a call for contributions with a link to the repo. In https://github.com/huggingface/hub-docs/blob/main/tasks/src/tasksData.ts#L23-L62 we would then have all the new types mapping to that default task page. That way we don't have to copy the content across 20-30 different files and will make it easier to maintain/update

**Comment by merveenoyan** [comment]
@osanseviero I created a placeholder file and also a contribution guide.

File: tasks-contribution-guide.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Contributing to Tasks
+
+Welcome to contribution guide to [Hugging Face Tasks](https://huggingface.co/tasks) and thank you for considering to help us out! 
+
+### Philosophy behind Tasks
+
+Task pages are made to lower the barrier of entry to understand a task that can be solved with machine learning and use or train a model to accomplish it. It's a documentation effort made to help out software developers, social scientists or anyone with no background in machine learning that is willing to use machine learning models to solve a problem. We tried to avoid fancy terminology to let everyone understand the documentation, and if terminology is needed, we tried to explain it on the most basic level that we could. This is important to understand before contributing to Tasks: at the end of every task page, we expect the user to be able to find and pull a model from the Hub and use it on their own data, see if it works for their use case to come up with a proof of concept.
```
Comment (osanseviero) [inline tasks-contribution-guide.md:7]:
```suggestion
The Task pages are made to lower the barrier of entry to understand a task that can be solved with machine learning and use or train a model to accomplish it. It's a documentation effort made to help out software developers, social scientists, or anyone with no background in machine learning that is interested in understanding how machine learning models can be used to solve a problem. The task pages avoid jargon to let everyone understand the documentation, and if specific terminology is needed, it is explained on the most basic level possible. This is important to understand before contributing to Tasks: at the end of every task page, the user is expected to be able to find and pull a model from the Hub and use it on their data and see if it works for their use case to come up with a proof of concept.
```

File: tasks-contribution-guide.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+### How to Contribute
+You can open a pull request to hub-docs repository to contribute a new documentation about a new task. Under `tasks/src` we have a folder for every task that contains two files, `about.md` and `data.ts`. `about.md` contains the markdown part of the page, use cases, resources and minimal code block to infer a model that belongs to the task. `data.ts` contains redirections to canonical models and datasets, metrics, the schema of the task and the information the inference widget needs.
```
Comment (osanseviero) [inline tasks-contribution-guide.md:10]: Maybe add a link to the hub-docs repository

File: tasks-contribution-guide.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+### How to Contribute
+You can open a pull request to hub-docs repository to contribute a new documentation about a new task. Under `tasks/src` we have a folder for every task that contains two files, `about.md` and `data.ts`. `about.md` contains the markdown part of the page, use cases, resources and minimal code block to infer a model that belongs to the task. `data.ts` contains redirections to canonical models and datasets, metrics, the schema of the task and the information the inference widget needs. 
+![Anatomy of a Task Page](tasks/assets/contribution-guide/anatomy.png)
+We have `tasks/assets` that contains data used in inference widget and images used in the markdown file. Last three files are `const.ts` which has task to library mapping (e.g. spacy to token-classification) where you can add a library. They will look on top right corner like below.
```
Comment (osanseviero) [inline tasks-contribution-guide.md:12]:
```suggestion
We have `tasks/assets` that contains data used in the inference widget and images used in the markdown file. The last file is `const.ts`, which has the task to library mapping (e.g. spacy to token-classification) where you can add a library. They will look in the top right corner like below.
```

File: tasks-contribution-guide.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+### How to Contribute
+You can open a pull request to hub-docs repository to contribute a new documentation about a new task. Under `tasks/src` we have a folder for every task that contains two files, `about.md` and `data.ts`. `about.md` contains the markdown part of the page, use cases, resources and minimal code block to infer a model that belongs to the task. `data.ts` contains redirections to canonical models and datasets, metrics, the schema of the task and the information the inference widget needs. 
+![Anatomy of a Task Page](tasks/assets/contribution-guide/anatomy.png)
```
Comment (osanseviero) [inline tasks-contribution-guide.md:11]: Nice image!

File: tasks/src/tabular-classification/about.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+### Fraud Detection
+Tabular classification models can be used in detecting fraudulent credit card transactions, where features can be amount of the transaction and account balance, and target to predict is whether the transaction is fraudulent or not. 
+
+### Churn Prediction
+Tabular classification models can be used in predicting customer churn in telecommunication. An example dataset for the task is hosted [here](https://huggingface.co/datasets/scikit-learn/churn-prediction).
+
+## Model Hosting and Inference
+
+You can use [Skops](https://github.com/skops-dev/skops) library to host `tabular-classification` models on Hugging Face Hub. With `Skops`, you can automatically generate configuration for your model that includes the information needed to enable the widget on Hugging Face Hub and requirements needed to infer the model. `Skops` also generates a model card including hyperparameter table and plot of the model, that you can later add information programmatically. You can push the model and the related files using Skops. The repository contains configuration file, model card and the model. You can pull a `scikit-learn` `tabular-classification` model like below:
```
Comment (osanseviero) [inline tasks/src/tabular-classification/about.md:11]:
```suggestion
You can use the [Skops](https://github.com/skops-dev/skops) library to host `scikit-learn` models on Hugging Face Hub. With `Skops`, you can automatically generate a configuration for your model that includes the information needed to enable the widget on Hugging Face Hub and the requirements needed to infer the model. `Skops` also generates a model card including a hyperparameter table and plot of the model, that you can later add information programmatically. You can push the model and the related files using Skops. The repository contains the configuration file, model card and the model. You can pull a `scikit-learn` `tabular-classification` model like below:
```

File: tasks/src/tabular-classification/about.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+hub_utils.download(repo_id="user-name/my-awesome-model", dst=target_path)
+model = joblib.load(Path(target_path)/"model.pkl")
+model.predict(sample)
+```
+
+
+## Useful Resources
+- Check out [scikit-learn organization](https://huggingface.co/scikit-learn) to learn more about different algorithms used for this task.
+- [Skops documentation](https://skops.readthedocs.io/en/latest/)
```
Comment (osanseviero) [inline tasks/src/tabular-classification/about.md:26]: Maybe also worth linking the blog post

File: tasks/src/tabular-classification/about.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+### Training your own model in just a few seconds
+
+We have built a [baseline trainer](https://huggingface.co/spaces/scikit-learn/baseline-trainer) application that you can drag and drop your dataset to, and it will train a baseline and push it to your Hugging Face Hub profile with automatically generated model card.
```
Comment (osanseviero) [inline tasks/src/tabular-classification/about.md:30]: Cool idea!

File: tasks/src/tabular-classification/data.ts [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+		{
+			description: "Binary classification dataset based on a census on income.",
+			id:          "scikit-learn/adult-census-income",
+		},
+		{
+			description: "Multi-class dataset on iris flower species.",
+			id:          "scikit-learn/iris",
+		},
+	],
+	demo: {
```
Comment (osanseviero) [inline tasks/src/tabular-classification/data.ts:17]: How will this show in the UI?

File: tasks/src/tabular-classification/data.ts [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+		},
+
+	],
+	models: [
+		{
+			description: "Breast cancer prediction model based on decision trees.",
+			id:          "scikit-learn/cancer-prediction-trees",
+		},
+	],
+	summary:      "Tabular classification is the task of classifying a target category based on a set of numerical or categorical features.",
```
Comment (osanseviero) [inline tasks/src/tabular-classification/data.ts:49]: For a non ML audience, they will not know what "categorical" or "features" means.

**Review by osanseviero — COMMENTED** [review]
Thanks a lot! This is very cool!

File: tasks/src/tabular-classification/data.ts (hunk 8 above)
Comment (merveenoyan) [reply tasks/src/tabular-classification/data.ts:17]: I need to run this by @beurkinger or @gary149 😅

File: tasks/src/tabular-classification/data.ts (hunk 9 above)
Comment (merveenoyan) [reply tasks/src/tabular-classification/data.ts:49]: I honestly don't know how I can make it more ELI5 :(

File: tasks/src/tabular-classification/about.md (hunk 6 above)
Comment (merveenoyan) [reply tasks/src/tabular-classification/about.md:26]: Addressed this.

File: tasks-contribution-guide.md (hunk 2 above)
Comment (merveenoyan) [reply tasks-contribution-guide.md]: Added!

**Comment by merveenoyan** [comment]
Closing this PR in favor of #303 and #302
